### YamlMime:UniversalReference
api_name: []
items:
- children:
  - azure.mgmt.storage.v2018_11_01.models.Reason.ACCOUNT_NAME_INVALID
  - azure.mgmt.storage.v2018_11_01.models.Reason.ALREADY_EXISTS
  class: azure.mgmt.storage.v2018_11_01.models.Reason
  fullName: azure.mgmt.storage.v2018_11_01.models.Reason
  inheritance:
  - inheritance:
    - type: builtins.object
    type: builtins.str
  - inheritance:
    - type: builtins.object
    type: enum.Enum
  langs:
  - python
  module: azure.mgmt.storage.v2018_11_01.models
  name: Reason
  summary: 'Gets the reason that a storage account name could not be used. The Reason
    element is only

    returned if NameAvailable is false.'
  syntax: {}
  type: class
  uid: azure.mgmt.storage.v2018_11_01.models.Reason
- class: azure.mgmt.storage.v2018_11_01.models.Reason
  fullName: azure.mgmt.storage.v2018_11_01.models.Reason.ACCOUNT_NAME_INVALID
  langs:
  - python
  module: azure.mgmt.storage.v2018_11_01.models
  name: ACCOUNT_NAME_INVALID
  syntax:
    content: ACCOUNT_NAME_INVALID = 'AccountNameInvalid'
  type: attribute
  uid: azure.mgmt.storage.v2018_11_01.models.Reason.ACCOUNT_NAME_INVALID
- class: azure.mgmt.storage.v2018_11_01.models.Reason
  fullName: azure.mgmt.storage.v2018_11_01.models.Reason.ALREADY_EXISTS
  langs:
  - python
  module: azure.mgmt.storage.v2018_11_01.models
  name: ALREADY_EXISTS
  syntax:
    content: ALREADY_EXISTS = 'AlreadyExists'
  type: attribute
  uid: azure.mgmt.storage.v2018_11_01.models.Reason.ALREADY_EXISTS
references:
- fullName: azure.mgmt.storage.v2018_11_01.models.Reason.ACCOUNT_NAME_INVALID
  isExternal: false
  name: ACCOUNT_NAME_INVALID
  parent: azure.mgmt.storage.v2018_11_01.models.Reason
  uid: azure.mgmt.storage.v2018_11_01.models.Reason.ACCOUNT_NAME_INVALID
- fullName: azure.mgmt.storage.v2018_11_01.models.Reason.ALREADY_EXISTS
  isExternal: false
  name: ALREADY_EXISTS
  parent: azure.mgmt.storage.v2018_11_01.models.Reason
  uid: azure.mgmt.storage.v2018_11_01.models.Reason.ALREADY_EXISTS
